pokemongo_bot/cell_workers/pokemon_optimizer.py

```diff
@@ -508,8 +508,8 @@ def get_evolution_plan(self, family_id, family_list, keep, try_evolve, try_upgra
             full_upgrade_candy_cost = 0
             full_upgrade_stardust_cost = 0
 
-            for i in range(pokemon.level, upgrade_level, 0.5):
-                upgrade_cost = self.pokemon_upgrade_cost[2 * (i - 1)]
+            for i in range(int(pokemon.level * 2), int(upgrade_level * 2)):
+                upgrade_cost = self.pokemon_upgrade_cost[i - 2]
```

Comment:
what kind of crash is it fixing? I don't see any difference between old and new code - it looks exactly the same (steps are twice as big, but "*2" is omitted when the couner variable is used, so...)

Can you show the logfile with the crash?